### PR TITLE
Add features tags for Atomics and SharedArrayBuffer

### PR DESCRIPTION
**Pull request by leobalter**
add features tags on files missing in the Atomics and SharedArrayBuffer folders

### CODE REVIEW

File: test/built-ins/Atomics/add/shared-nonint-views.js [hunk 1]
```diff
@@ -6,7 +6,7 @@ esid: sec-atomics.add
 description: >
   Test Atomics.add on shared non-integer TypedArrays
 includes: [testTypedArray.js]
-features: [TypedArray]
+features: [Atomics, TypedArray]
 ---*/
 
 var sab = new SharedArrayBuffer(1024);
```
Comment (nomadtechie) [inline test/built-ins/Atomics/add/shared-nonint-views.js]: missing SharedArrayBuffer

File: test/built-ins/Atomics/exchange/shared-nonint-views.js [hunk 2]
```diff
@@ -6,7 +6,7 @@ esid: sec-atomics.exchange
 description: >
   Test Atomics.exchange on shared non-integer TypedArrays
 includes: [testTypedArray.js]
-features: [TypedArray]
+features: [Atomics, TypedArray]
```
Comment (nomadtechie) [inline test/built-ins/Atomics/exchange/shared-nonint-views.js]: missing SharedArrayBuffer

File: test/built-ins/Atomics/exchange/nonshared-int-views.js [hunk 3]
```diff
@@ -6,7 +6,7 @@ esid: sec-atomics.exchange
 description: >
   Test Atomics.exchange on non-shared integer TypedArrays
 includes: [testTypedArray.js]
-features: [TypedArray]
+features: [Atomics, TypedArray]
```
Comment (nomadtechie) [inline test/built-ins/Atomics/exchange/nonshared-int-views.js]: missing ArrayBuffer

File: test/built-ins/Atomics/compareExchange/nonshared-int-views.js [hunk 4]
```diff
@@ -6,7 +6,7 @@ esid: sec-atomics.compareexchange
 description: >
   Test Atomics.compareExchange on non-shared integer TypedArrays
 includes: [testTypedArray.js]
-features: [TypedArray]
+features: [Atomics, TypedArray]
```
Comment (nomadtechie) [inline test/built-ins/Atomics/compareExchange/nonshared-int-views.js]: missing ArrayBuffer

File: test/built-ins/Atomics/and/nonshared-int-views.js [hunk 5]
```diff
@@ -6,7 +6,7 @@ esid: sec-atomics.and
 description: >
   Test Atomics.and on non-shared integer TypedArrays
 includes: [testTypedArray.js]
-features: [TypedArray]
+features: [Atomics, TypedArray]
```
Comment (nomadtechie) [inline test/built-ins/Atomics/and/nonshared-int-views.js]: missing ArrayBuffer

File: test/built-ins/Atomics/add/nonshared-int-views.js [hunk 6]
```diff
@@ -6,7 +6,7 @@ esid: sec-atomics.add
 description: >
   Test Atomics.add on non-shared integer TypedArrays
 includes: [testTypedArray.js]
-features: [TypedArray]
+features: [Atomics, TypedArray]
 ---*/
 
 var ab = new ArrayBuffer(16);
```
Comment (nomadtechie) [inline test/built-ins/Atomics/add/nonshared-int-views.js]: missing ArrayBuffer

File: test/built-ins/Atomics/load/nonshared-int-views.js [hunk 7]
```diff
@@ -6,7 +6,7 @@ esid: sec-atomics.load
 description: >
   Test Atomics.load on non-shared integer TypedArrays
 includes: [testTypedArray.js]
-features: [TypedArray]
+features: [Atomics, TypedArray]
```
Comment (nomadtechie) [inline test/built-ins/Atomics/load/nonshared-int-views.js]: missing ArrayBuffer

File: test/built-ins/Atomics/or/nonshared-int-views.js [hunk 8]
```diff
@@ -6,7 +6,7 @@ esid: sec-atomics.or
 description: >
   Test Atomics.or on non-shared integer TypedArrays
 includes: [testTypedArray.js]
-features: [TypedArray]
+features: [Atomics, TypedArray]
```
Comment (nomadtechie) [inline test/built-ins/Atomics/or/nonshared-int-views.js]: missing ArrayBuffer

File: test/built-ins/Atomics/wait/nonshared-int-views.js [hunk 9]
```diff
@@ -6,7 +6,7 @@ esid: sec-atomics.wait
 description: >
   Test Atomics.wait on non-shared integer TypedArrays
 includes: [testTypedArray.js]
-features: [TypedArray]
+features: [Atomics, TypedArray]
```
Comment (nomadtechie) [inline test/built-ins/Atomics/wait/nonshared-int-views.js]: missing ArrayBuffer

File: test/built-ins/Atomics/xor/nonshared-int-views.js [hunk 10]
```diff
@@ -6,7 +6,7 @@ esid: sec-atomics.xor
 description: >
   Test Atomics.xor on non-shared integer TypedArrays
 includes: [testTypedArray.js]
-features: [TypedArray]
+features: [Atomics, TypedArray]
 ---*/
```
Comment (nomadtechie) [inline test/built-ins/Atomics/xor/nonshared-int-views.js]: missing ArrayBuffer

File: test/built-ins/SharedArrayBuffer/prototype/byteLength/this-is-arraybuffer.js [hunk 11]
```diff
@@ -4,6 +4,7 @@
 /*---
 esid: sec-get-sharedarraybuffer.prototype.bytelength
 description: Throws a TypeError exception when `this` is an ArrayBuffer
+features: [SharedArrayBuffer]
 ---*/
```
Comment (nomadtechie) [inline test/built-ins/SharedArrayBuffer/prototype/byteLength/this-is-arraybuffer.js]: missing ArrayBuffer

File: test/built-ins/SharedArrayBuffer/prototype/slice/nonconstructor.js [hunk 12]
```diff
@@ -12,6 +12,7 @@ info: |
     Built-in function objects that are not identified as constructors do not
     implement the [[Construct]] internal method unless otherwise specified
     in the description of a particular function.
+features: [SharedArrayBuffer]
```
Comment (nomadtechie) [inline test/built-ins/SharedArrayBuffer/prototype/slice/nonconstructor.js]: missing ArrayBuffer

File: test/built-ins/SharedArrayBuffer/prototype/slice/this-is-arraybuffer.js [hunk 13]
```diff
@@ -5,6 +5,7 @@
 esid: sec-sharedarraybuffer.prototype.slice
 description: >
   Throws a TypeError if `this` is an ArrayBuffer
+features: [SharedArrayBuffer]
 ---*/
```
Comment (nomadtechie) [inline test/built-ins/SharedArrayBuffer/prototype/slice/this-is-arraybuffer.js]: ArrayBuffer

File: test/built-ins/Atomics/and/shared-nonint-views.js [hunk 14]
```diff
@@ -6,7 +6,7 @@ esid: sec-atomics.and
 description: >
   Test Atomics.and on shared non-integer TypedArrays
 includes: [testTypedArray.js]
-features: [TypedArray]
+features: [Atomics, TypedArray]
```
Comment (nomadtechie) [inline test/built-ins/Atomics/and/shared-nonint-views.js]: missing SharedArrayBuffer

File: test/built-ins/Atomics/load/shared-nonint-views.js [hunk 15]
```diff
@@ -6,7 +6,7 @@ esid: sec-atomics.load
 description: >
   Test Atomics.load on shared non-integer TypedArrays
 includes: [testTypedArray.js]
-features: [TypedArray]
+features: [Atomics, TypedArray]
```
Comment (nomadtechie) [inline test/built-ins/Atomics/load/shared-nonint-views.js]: missing SharedArrayBuffer

File: test/built-ins/Atomics/or/shared-nonint-views.js [hunk 16]
```diff
@@ -6,7 +6,7 @@ esid: sec-atomics.or
 description: >
   Test Atomics.or on shared non-integer TypedArrays
 includes: [testTypedArray.js]
-features: [TypedArray]
+features: [Atomics, TypedArray]
```
Comment (nomadtechie) [inline test/built-ins/Atomics/or/shared-nonint-views.js]: missing SharedArrayBuffer

File: test/built-ins/Atomics/store/nonshared-int-views.js [hunk 17]
```diff
@@ -6,7 +6,7 @@ esid: sec-atomics.store
 description: >
   Test Atomics.store on non-shared integer TypedArrays
 includes: [testTypedArray.js]
-features: [TypedArray]
+features: [Atomics, TypedArray]
```
Comment (nomadtechie) [inline test/built-ins/Atomics/store/nonshared-int-views.js]: missing ArrayBuffer

File: test/built-ins/Atomics/store/shared-nonint-views.js [hunk 18]
```diff
@@ -6,7 +6,7 @@ esid: sec-atomics.store
 description: >
   Test Atomics.store on shared non-integer TypedArrays
 includes: [testTypedArray.js]
-features: [TypedArray]
+features: [Atomics, TypedArray]
```
Comment (nomadtechie) [inline test/built-ins/Atomics/store/shared-nonint-views.js]: missing SharedArrayBuffer

File: test/built-ins/Atomics/sub/nonshared-int-views.js [hunk 19]
```diff
@@ -6,7 +6,7 @@ esid: sec-atomics.sub
 description: >
   Test Atomics.sub on non-shared integer TypedArrays
 includes: [testTypedArray.js]
-features: [TypedArray]
+features: [Atomics, TypedArray]
```
Comment (nomadtechie) [inline test/built-ins/Atomics/sub/nonshared-int-views.js]: missing ArrayBuffer

File: test/built-ins/Atomics/sub/shared-nonint-views.js [hunk 20]
```diff
@@ -6,7 +6,7 @@ esid: sec-atomics.sub
 description: >
   Test Atomics.sub on shared non-integer TypedArrays
 includes: [testTypedArray.js]
-features: [TypedArray]
+features: [Atomics, TypedArray]
```
Comment (nomadtechie) [inline test/built-ins/Atomics/sub/shared-nonint-views.js]: missing SharedArrayBuffer

File: test/built-ins/Atomics/wait/did-timeout.js [hunk 21]
```diff
@@ -7,6 +7,7 @@ description: >
   Test that Atomics.wait returns the right result when it timed out and that
   the time to time out is reasonable.
 includes: [atomicsHelper.js]
+features: [Atomics]
```
Comment (nomadtechie) [inline test/built-ins/Atomics/wait/did-timeout.js]: missing SharedArrayBuffer, TypedArray

File: test/built-ins/Atomics/wait/good-views.js [hunk 22]
```diff
@@ -6,6 +6,7 @@ esid: sec-atomics.wait
 description: >
   Test Atomics.wait on arrays that allow atomic operations,
   in an Agent that is allowed to wait.
+features: [Atomics]
```
Comment (nomadtechie) [inline test/built-ins/Atomics/wait/good-views.js]: missing SharedArrayBuffer, TypedArray, ArrayBuffer

File: test/built-ins/Atomics/wait/nan-timeout.js [hunk 23]
```diff
@@ -5,6 +5,7 @@
 esid: sec-atomics.wait
 description: >
   Test that Atomics.wait does not time out with a NaN timeout
+features: [Atomics]
```
Comment (nomadtechie) [inline test/built-ins/Atomics/wait/nan-timeout.js]: missing SharedArrayBuffer, TypedArray

File: test/built-ins/Atomics/wait/negative-timeout.js [hunk 24]
```diff
@@ -5,6 +5,7 @@
 esid: sec-atomics.wait
 description: >
   Test that Atomics.wait times out with a negative timeout
+features: [Atomics]
```
Comment (nomadtechie) [inline test/built-ins/Atomics/wait/negative-timeout.js]: missing SharedArrayBuffer, TypedArray

File: test/built-ins/Atomics/wait/no-spurious-wakeup.js [hunk 25]
```diff
@@ -7,6 +7,7 @@ description: >
   Test that Atomics.wait actually waits and does not spuriously wake
   up when the memory value is changed.
 includes: [atomicsHelper.js]
+features: [Atomics]
```
Comment (nomadtechie) [inline test/built-ins/Atomics/wait/no-spurious-wakeup.js]: missing SharedArrayBuffer, TypedArray

File: test/built-ins/Atomics/wait/shared-nonint-views.js [hunk 26]
```diff
@@ -6,7 +6,7 @@ esid: sec-atomics.wait
 description: >
   Test Atomics.wait on shared non-integer TypedArrays
 includes: [testTypedArray.js]
-features: [TypedArray]
+features: [Atomics, TypedArray]
```
Comment (nomadtechie) [inline test/built-ins/Atomics/wait/shared-nonint-views.js]: missing SharedArrayBuffer

File: test/built-ins/Atomics/wait/was-woken.js [hunk 27]
```diff
@@ -5,6 +5,7 @@
 esid: sec-atomics.wait
 description: >
   Test that Atomics.wait returns the right result when it was awoken.
+features: [Atomics]
```
Comment (nomadtechie) [inline test/built-ins/Atomics/wait/was-woken.js]: missing SharedArrayBuffer, TypedArray

File: test/built-ins/Atomics/wake/counts.js [hunk 28]
```diff
@@ -5,6 +5,7 @@
 esid: sec-atomics.wake
 description: >
   Allowed boundary cases of the third 'count' argument to Atomics.wake
+features: [Atomics]
```
Comment (nomadtechie) [inline test/built-ins/Atomics/wake/counts.js]: missing SharedArrayBuffer, TypedArray

File: test/built-ins/Atomics/wake/nonshared-int-views.js [hunk 29]
```diff
@@ -6,7 +6,7 @@ esid: sec-atomics.wake
 description: >
   Test Atomics.wake on non-shared integer TypedArrays
 includes: [testTypedArray.js]
-features: [TypedArray]
+features: [Atomics, TypedArray]
```
Comment (nomadtechie) [inline test/built-ins/Atomics/wake/nonshared-int-views.js]: missing ArrayBuffer, TypedArray

File: test/built-ins/Atomics/wake/shared-nonint-views.js [hunk 30]
```diff
@@ -6,7 +6,7 @@ esid: sec-atomics.wake
 description: >
   Test Atomics.wake on shared non-integer TypedArrays
 includes: [testTypedArray.js]
-features: [TypedArray]
+features: [Atomics, TypedArray]
```
Comment (nomadtechie) [inline test/built-ins/Atomics/wake/shared-nonint-views.js]: missing SharedArrayBuffer

File: test/built-ins/Atomics/wake/wake-all-on-loc.js [hunk 31]
```diff
@@ -6,6 +6,7 @@ esid: sec-atomics.wake
 description: >
   Test that Atomics.wake wakes all waiters on a location, but does not
   wake waiters on other locations.
+features: [Atomics]
```
Comment (nomadtechie) [inline test/built-ins/Atomics/wake/wake-all-on-loc.js]: missing SharedArrayBuffer, TypedArray

File: test/built-ins/Atomics/wake/wake-all.js [hunk 32]
```diff
@@ -5,6 +5,7 @@
 esid: sec-atomics.wake
 description: >
   Test that Atomics.wake wakes all waiters if that's what the count is.
+features: [Atomics]
```
Comment (nomadtechie) [inline test/built-ins/Atomics/wake/wake-all.js]: missing SharedArrayBuffer, TypedArray

File: test/built-ins/Atomics/wake/wake-in-order.js [hunk 33]
```diff
@@ -5,6 +5,7 @@
 esid: sec-atomics.wake
 description: >
   Test that Atomics.wake wakes agents in the order they are waiting.
+features: [Atomics]
```
Comment (nomadtechie) [inline test/built-ins/Atomics/wake/wake-in-order.js]: missing SharedArrayBuffer, TypedArray

File: test/built-ins/Atomics/wake/wake-nan.js [hunk 34]
```diff
@@ -5,6 +5,7 @@
 esid: sec-atomics.wake
 description: >
   Test that Atomics.wake wakes zero waiters if the count is NaN
+features: [Atomics]
```
Comment (nomadtechie) [inline test/built-ins/Atomics/wake/wake-nan.js]: missing SharedArrayBuffer, TypedArray

File: test/built-ins/Atomics/wake/wake-negative.js [hunk 35]
```diff
@@ -5,6 +5,7 @@
 esid: sec-atomics.wake
 description: >
   Test that Atomics.wake wakes zero waiters if the count is negative
+features: [Atomics]
```
Comment (nomadtechie) [inline test/built-ins/Atomics/wake/wake-negative.js]: missing SharedArrayBuffer, TypedArray

File: test/built-ins/Atomics/wake/wake-one.js [hunk 36]
```diff
@@ -5,6 +5,7 @@
 esid: sec-atomics.wake
 description: >
   Test that Atomics.wake wakes one waiter if that's what the count is.
+features: [Atomics]
```
Comment (nomadtechie) [inline test/built-ins/Atomics/wake/wake-one.js]: missing SharedArrayBuffer, TypedArray

File: test/built-ins/Atomics/wake/wake-two.js [hunk 37]
```diff
@@ -5,6 +5,7 @@
 esid: sec-atomics.wake
 description: >
   Test that Atomics.wake wakes two waiters if that's what the count is.
+features: [Atomics]
```
Comment (nomadtechie) [inline test/built-ins/Atomics/wake/wake-two.js]: missing SharedArrayBuffer, TypedArray

File: test/built-ins/Atomics/wake/wake-zero.js [hunk 38]
```diff
@@ -5,6 +5,7 @@
 esid: sec-atomics.wake
 description: >
   Test that Atomics.wake wakes zero waiters if that's what the count is.
+features: [Atomics]
```
Comment (nomadtechie) [inline test/built-ins/Atomics/wake/wake-zero.js]: missing SharedArrayBuffer, TypedArray

File: test/built-ins/Atomics/xor/shared-nonint-views.js [hunk 39]
```diff
@@ -6,7 +6,7 @@ esid: sec-atomics.xor
 description: >
   Test Atomics.xor on shared non-integer TypedArrays
 includes: [testTypedArray.js]
-features: [TypedArray]
+features: [Atomics, TypedArray]
```
Comment (nomadtechie) [inline test/built-ins/Atomics/xor/shared-nonint-views.js]: missing SharedArrayBuffer

**Comment by rwaldron** [comment]
@leobalter check the conflict please 👍

**Comment by leobalter** [comment]
While I'm resolving the conflict, my idea was to fill the tests only with features for their respective folder. I still need to do some follow up to fix with other missing tags as I imagine we might have other files missing some tags as well.

**Pull request merged**